src/client.js

```diff
@@ -1654,16 +1654,41 @@ class Client {
     async getSchema(schemaId, representation, internal) {
         var entity = await this._entityCache.get("xtk:schema", schemaId);
         if (!entity) {
-          entity = await this.getEntityIfMoreRecent("xtk:schema", schemaId, "xml", internal);
-          if (entity) {
-            const impls = DomUtil.getAttributeAsString(entity, "implements");
-            if (impls === "xtk:persist" && schemaId !== "xtk:session" && schemaId !== "xtk:persist") {
-                // Ensure xtk:persist is present by loading the xtk:session schema
-                await this.getSchema("xtk:session", "xml", true);
+              // special case of temp schema for nms:group
+              if (schemaId.startsWith("temp:group:")) {
+                  var parts = schemaId.split(":");
+                  var queryDef = {
+                    "schema": "nms:group",
+                    "operation": "get",
+                    "select": {
+                      "node": [
+                        { "expr": "@id" },
+                        { "expr": "extension" }
+                      ]
+                    },
+                    "where": {
+                      "condition": [
+                        { "expr": "@id=" + parts[2] }
+                      ]
+                    }
+                  };
+                  // Convert to current representation
+                  queryDef = this._convertToRepresentation(queryDef, "SimpleJson");
+                  const query = this.NLWS.xtkQueryDef.create(queryDef);
+                  const groupSchema = await query.executeQuery();
```

Comment:
The corresponding SOAP call should have "internal" flag set as this query is performed under the hood by the SDK